app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.cdnfonts.com/css/gamelan');
+
+* {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'gamelan';
+}
+
+.category_show {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  margin: 0.7rem auto;
+  box-shadow: rgba(255, 238, 4, 0.05) 0 1px 0, rgba(17, 17, 26, 0.1) 0 0 8px;
+  width: 90%;
+  color: #000;
+}
+
+a {
+  text-decoration: none;
+}
+
+.category_show_content > img {
+  border-radius: 100%;
+  width: 5rem;
+  height: 5rem;
+  border: 2px solid #ffc107;
+}
+
+.category_show > img {
+  border-radius: 100%;
+  width: 5rem;
+  height: 5rem;
+  margin: auto;
+}
+
+p {
+  font-size: 13px;
+}
+
+.navbar-dark .navbar-brand,
+.navbar-dark .navbar-toggler {
+  color: #ffc107;
```

Comment:
* Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) as stated in [the requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.) , including:

> Colors.
> Typography: font face, size and weight.
> Layout: composition and space between elements.